### PR TITLE
Update docker mvc migration docker image url

### PR DESCRIPTION
**Pull request by synercoder**
The current code in the documentation doesn't work. After a bit of digging I found https://hub.docker.com/r/microsoft/dotnet-framework/ and https://github.com/microsoft/dotnet-framework-docker/blob/main/samples/aspnetmvcapp/Dockerfile as an example.

Based on those I updated the text and example dockerfile.

### CODE REVIEW

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 1]
```diff
@@ -69,17 +69,17 @@ Click **Publish**, and Visual Studio will copy all the needed assets to the dest
 Create a new file named *Dockerfile* to define your Docker image. *Dockerfile* contains instructions to build the final image and includes any base image names, required components, the app you want to run, and other configuration images. *Dockerfile* is the input to the `docker build` command that creates the image.
 
 For this exercise, you will build an image based on the `microsoft/aspnet`
-image located on [Docker Hub](https://hub.docker.com/r/microsoft/aspnet/).
-The base image, `microsoft/aspnet`, is a Windows Server image. It contains
-Windows Server Core, IIS, and ASP.NET 4.7.2. When you run this image in your container, it will automatically start IIS and installed websites.
+image located on [Docker Hub](https://hub.docker.com/r/microsoft/dotnet-framework/).
```
Comment (Rick-Anderson) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md:72]: Where's the image? go to https://hub.docker.com/r/microsoft/dotnet-framework/ and select the link [dotnet/framework/aspnet](https://hub.docker.com/_/microsoft-dotnet-framework-aspnet/): ASP.NET Web Forms and MVC
and the link doesn't work

Comment (Rick-Anderson) [reply aspnet/mvc/overview/deployment/docker-aspnetmvc.md:72]: The correct link is https://hub.docker.com/r/microsoft/dotnet-framework-aspnet I think. The have `/_/` when the should have `/r/`. Can you add that link?

Comment (synercoder) [reply aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: Updated the link to the suggested correction.

**Comment by Rick-Anderson** [comment]
@synercoder  thanks for the PR. Hope to see you on the ASP.NET Core repo.

**Pull request merged**